### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by sconde** [jorge3fernandes/City_of_Brockton_R#5]
lets add more info in the README file

**Comment by sconde** [comment]
@jorge3fernandes anymore edits needed for this?

**Comment by jorge3fernandes** [comment]
yes. We have sections that are still empty. I'll look at it and see what I can add. Feel free to add as well.
